src/create_helpers.ts

```diff
@@ -67,24 +67,89 @@ function validateModuleChildren(children?: ts.Node[])
     }
 }
 
+function formatMultilineComment(comment: string): string {
+    return comment.split('\n').join('\n * ');
+}
+
+function handlePropsComment(props: IDocletProp[], jsdocTagName: String): string[] {
+    return props.map(prop => {
+        if (prop.description) {
+            let name;
+            if (prop.optional) {
+                if (prop.defaultvalue !== undefined) {
+                    name = `[${prop.name} = ${prop.defaultvalue}]`;
+                } else {
+                    name = `[${prop.name}]`;
+                }
+            } else {
+                name = prop.name;
+            }
+            const description = ` - ${formatMultilineComment(prop.description)}`;
+            return `\n * @${jsdocTagName} ${name}${description}`
+        }
+        return ''
+    }).filter(value => value !== '')
```

Comment:
I'd just do the `join('')` here since all usages of this function do it later anyway. Should make the combining string later more readable to skip the join.

Reply:
Good point, I fixed it.